src/components/Paginator.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+
+class Paginator extends React.Component {
+  nextClick(e) {
+    e.preventDefault()
+
+    if (this.props.nextURL != null) {
+      this.props.loadPlaylists(this.props.nextURL)
+    }
+  }
+
+  prevClick(e) {
+    e.preventDefault()
+
+    if (this.props.prevURL != null) {
+      this.props.loadPlaylists(this.props.prevURL)
+    }
+  }
+
+  render() {
+    if (this.props.nextURL != null || this.props.prevURL != null) {
+      return (
+        <nav className="paginator text-right">
+          <ul className="pagination pagination-sm">
+            <li className={this.props.prevURL == null ? 'disabled' : ''}>
+              { /* eslint-disable-next-line  */ }
+              <a href="#" aria-label="Previous" onClick={this.prevClick.bind(this)}>
+                <span aria-hidden="true">&laquo;</span>
+              </a>
+            </li>
+            <li className={this.props.nextURL == null ? 'disabled' : ''}>
+              { /* eslint-disable-next-line  */ }
+              <a href="#" aria-label="Next" onClick={this.nextClick.bind(this)}>
+                <span aria-hidden="true">&raquo;</span>
+              </a>
+            </li>
+          </ul>
+        </nav>
+      )
+    } else {
+      return <div>&nbsp;</div>
```

Comment:
it's a little bit odd to return an empty `div`
Could `return null` do the job ? or at least `return <div></div>` ?